sharedUtils/getClientId.js

```diff
@@ -0,0 +1,33 @@
+import { getApplicationConfig } from './config'
+
+/**
+ * Replaces ENV and PORTAL in a clientId string
+ * @param {String} clientId
+ */
+const replaceClientIdEnvPortal = (clientId, envOverride) => (
+  clientId
+    .replace('ENV', envOverride || getApplicationConfig().env)
+    .replace('PORTAL', getApplicationConfig().defaultPortal)
+)
+
+/**
+ * Returns the clientId to send with CMR requests
+ */
+export const getClientId = () => {
+  // Override the Env with 'test' if we are in CI
+  const { ciMode, clientId } = getApplicationConfig()
```

Comment:
I'd like to eventually figure out a way to not need this `ciMode` config -- we should be able to derive this or use another config.